modules/nw-kube-proxy-configuring.adoc

```diff
@@ -66,7 +66,7 @@ items:
     - cidr: 10.128.0.0/14
       hostPrefix: 23
     defaultNetwork:
-      type: OpenShiftSDN
+      type: OVN-Kubernetes
```

Comment:
OVNKubernetes network type doesn't use kube-proxy. If this particular section documents the kube-proxy configuration, we need to figure out where this should live but certainly it is incorrect to simply replace openshift-sdn with ovn-k here.

Reply:
cc @danwinship for input on what this section should be updated to.

Reply:
@abhat, this entire module is about _Modifying the kube-proxy configuration_, so for our purposes I think this just goes away?